### PR TITLE
Modify the loader to the current version of EvalAI

### PR DESCRIPTION
**Pull request by Sanji515**
#### Changes proposed in this pull request:

- Added the required html css code in the loader component.

- Link to live demo: http://pr-184-evalai.surge.sh

### CODE REVIEW

**Comment by Sanji515** [comment]
@lunayach Please review.

**Comment by sanketbansal** [comment]
@Sanji515 can you please refer #160.
Cherry pick the required commits.

File: src/app/components/utility/loading/loading.component.html [hunk 1]
```diff
@@ -1,7 +1,9 @@
-<div class="loading-container">
-<div class="loader" id="loader-4">
-          <span></span>
-          <span></span>
-          <span></span>
-</div>
+<div class="loader-container card-loader-container fade">
+    <div class="loader">
+        <div></div>
```
Comment (lunayach) [inline src/app/components/utility/loading/loading.component.html]:
```suggestion
        <div *ngFor="let i of [0,1,2,3,4]"> </div>
```

**Review by lunayach — COMMENTED** [review]
This Works!
But @RishabhJain2018, @sanketbansal had also pushed a PR for this (#160) Should we close that then?

**Comment by RishabhJain2018** [comment]
Yes, since you've already reviewed this one.

File: src/app/components/utility/loading/loading.component.html (hunk 1 above)
Comment (Sanji515) [reply src/app/components/utility/loading/loading.component.html]: @lunayach done

**Review by lunayach — APPROVED** [review]
Looks good to be merged.

**Pull request merged**